### DISCUSSION TITLE
Welcome to Instagram-Clone-SwiftUI Discussions!

### DISCUSSION THREAD

**Discussion by PankajGaikar** [PankajGaikar/Instagram-Clone-SwiftUI#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MishkaNadine** [comment]
Hi

Anyone that can help me out with getting code that will help me remove any possible hackers on my instagram account??

**Reply by PankajGaikar** [reply]
This is not the place for such discussion I believe.

**Reply by MishkaNadine** [reply]
could you perhaps contact me via email then ? it's in my readme file

**Reply by PankajGaikar** [reply]
What I suggested is this is a demo to showcase what I did with SwiftUI. This code has nothing to do with Instagram nor I can help you with what you are looking for. Sorry.
